lib/kb_mash/mash_utils/MashUtils.py

```diff
@@ -93,9 +110,42 @@ def sketch_service_query(self, input_upas, n_max_results, search_db):
                 }
             }
             print("Processing input: %s" % input_name)
-            resp = requests.post(url=sketch_url, data=json.dumps(payload),headers={
-                'content-type':"application/json-rpc",'Authorization':self.auth_token},
-                timeout=600)
+            attempts = 1
+            max_attempts = 5
+            while True:
+                # there's really no good way to test retries w/o a ton of work
+                start = time.time()
+                try:
+                    resp = requests.post(url=sketch_url, data=json.dumps(payload),headers={
```

Comment:
Pretty sure this line won't pass pep8, it's pretty squashed

Reply:
I didn't actually modify that line other than indenting it, but I'll clean it up